scripts/exrrfs_prep_lbc.sh

```diff
@@ -35,7 +35,7 @@ while [[ $n -le 12 ]]; do
   YYYYMMDDlbc=${CDATElbc:0:8}
   HHlbc=${CDATElbc:8:2}
   checkfile=${COMINrrfs}/${RUN}.${YYYYMMDDlbc}/${HHlbc}${MEMDIR}/lbc/${last_bdyfile}
-  if [ -r "${checkfile}" ];then
+  if [ -r ${checkfile} ];then
```

Comment:
suggest using double brackets. Add one space before `then`

```
if [[ -r ${checkfile} ]]; then
```

Also, I am curious why to remove the double quotes since using them is safer than without them.  I just did a quick test and it worked without a problem.

Reply:
The old line ` if [ -r "${checkfile}" ];  then` didn't work but if it works with double brackets let's do what you suggested.